### DISCUSSION TITLE
V2 - EXPO EAS problems

### DISCUSSION THREAD

**Discussion by wortkotze** [gluestack/gluestack-ui#2364] [Q&A]
Hi, I migrated our project to V2. after the migration we are running in now in problems with the EAS Expo Build service.

❌  error: global.css: A jest worker process (pid=4932) was terminated by another process: signal=SIGTERM, exitCode=null. Operating system logs may contain more information on why this occurred.

Does anyone had the same experience or probably already a solution?

thx
Stephan

**Comment by wortkotze** [comment]
no one else has this problem?

**Reply by brayohmurithidev** [reply]
I am facing the very same issue. Have you been able to solve this??

**Reply by wortkotze** [reply]
No, not until now.
I asked also in the discord support channel no reaction from gluestack team
